test/e2e/PISPTransferFailure.test.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * @name PISPTransferFailure
+ * @description PISP End To End Tests
+ */
+
+describe('PISP initiated transfer failure', (): void => {
+  describe('1. PISP GET /parties response', (): void => {
```

Comment:
Since `GET /parties` is a part of the Mojaloop API, we probably just want to cover 1 or 2 scenarios here. Let's keep lines 10, 11, 35, 36, 37

Reply:
done.